src/content/reference/react-dom/server/index.md

```diff
@@ -1,49 +1,48 @@
 ---
-title: Server React DOM APIs
+title: APIs de servidor do React DOM
 ---
 
 <Intro>
 
-The `react-dom/server` APIs let you render React components to HTML on the server. These APIs are only used on the server at the top level of your app to generate the initial HTML. A [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) may call them for you. Most of your components don't need to import or use them.
-
+As APIs `react-dom/server` permitem que você renderize componentes React em HTML no servidor. Essas APIs são usadas apenas no servidor na raiz de sua aplicação para gerar o HTML inicial. A [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) pode chamá-los para você. A maioria de seus componentes não precisa importar ou usá-los.
```

Comment:
Sobre a expressão em inglês, _at the top level_, ela pode ser traduzida como "no nível mais alto" ou "ao mais alto nível". Sendo assim, a tradução ficaria: **Essas APIs são usadas apenas no servidor no nível mais alto de sua aplicação para gerar o HTML inicial.**

Reply:
```suggestion
As APIs `react-dom/server` permitem que você renderize componentes React em HTML no servidor. Essas APIs são usadas apenas no servidor no nível mais alto de sua aplicação para gerar o HTML inicial. Um [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) pode chamá-los para você. A maioria de seus componentes não precisa importar ou usá-los.
```

Reply:
Particularmente entendo que a tradução "no nível mais alto" faz mais sentido na grande maioria das vezes quando queremos traduzir este termo 'at the top level' para o Português.